store/store.ts

```diff
@@ -113,6 +133,27 @@ export function createStore<S extends AnyState>({
     yield* next();
   }
 
+  if (determineNextState?.setup) determineNextState.setup();
+
+  function* updateMdw(ctx: UpdaterCtx<S>, next: Next) {
```

Comment:
This could be a mdw that users always provide instead of forcing it to be included in the store.  In that case this would reduce the default store functionality, just a thought.